indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -316,6 +316,18 @@ def _process(self):
         return redirect(url_for('.manage_regform_list', self.event))
 
 
+class RHRegistrationFormClone(RHManageRegFormBase):
+    """Clone a registration form."""
+
+    def _process(self):
+        form = clone_registration_form(self.regform)
+        flash(_('Registration form cloned. You can now modify it.'), 'success')
+        logger.info('Registration form %s cloned by %s', self.regform, session.user)
+        log_text = f'Registration form cloned from "{self.regform.title}"'
+        self.regform.log(EventLogRealm.managment, LogKind.positive, 'Registration', log_text, session.user)
```

Comment:
We could include the new regform id in the logs:
```suggestion
        self.regform.log(EventLogRealm.managment, LogKind.positive, 'Registration', log_text, session.user, meta={'new_registration_form_id': form.id})
```

Reply:
I'd rather use the new regform's log method so the log entry is associated with that one.